frontend/cypress/e2e/talentsearch/search-workflows.cy.js

```diff
@@ -8,89 +11,308 @@ describe("Talent Search Workflow Tests", () => {
-      cy.wait("@gqlCountApplicantsAndCountPoolCandidatesByPoolQuery");
+    // skills selection, not currently used in search
+    cy.get("@testSkill").then((skill) => {
+      cy.findByRole("button", {
+        name: `Add this skill : ${skill.name.en}`,
+      }).click();
+      // skill selection does not trigger an api request
+      searchFindsMySingleCandidate();
+    });
 
-    })
+    // conditions of employment, no negation possible
+    cy.findByRole("checkbox", {
+      name: /ability to work overtime \(Occasionally\)/i,
+    }).click();
+    cy.wait("@gqlCountApplicantsAndCountPoolCandidatesByPoolQuery");
+    searchFindsMySingleCandidate();
 
-    searchReturnsGreaterThanZeroApplicants();
+    // check total user count
+    // no way to know what the exact number should be without resetting the database, so just check it's not zero
+    cy.findByRole("heading", {
+      name: /Results: [1-9][0-9]* matching candidate/i,
+    });
 
-    cy.findAllByRole("button", { name: /Request Candidates/i }).then($button => {
-      cy.wrap($button)
-        .should("exist")
-        .and("be.visible")
-        .and("not.be.disabled");
+    cy.findAllByRole("article", {
+      name: `Cypress Test Pool EN 1 ${uniqueTestId} (IT-01 Business Line Advisory Services)`,
+    }).within(() => {
+      cy.findByRole("button", { name: /Request Candidates/i }).click();
     });
-    // Force click to mitigate async re-render of button that detaches its DOM element.
-    // See: https://github.com/GCTC-NTGC/gc-digital-talent/pull/4119#issuecomment-1271642887
-    cy.findAllByRole("button", { name: /Request Candidates/i }).first().click({ force: true });
 
+    /*
+     * Request Page (/en/search/request)
+     * I'm using findAllByText instead of findByText since the strings appear multiple times in the DOM.
+     */
     cy.wait("@gqlgetPoolCandidateSearchRequestDataQuery");
 
-    cy.findByRole("textbox", { name: /Full Name/i })
-      .type("Test Full Name");
+    cy.findByRole("textbox", { name: /Full Name/i }).type("Test Full Name");
 
-    cy.findByRole("textbox", { name: /Government e-mail/i })
-      .type("test@tbs-sct.gc.ca");
+    cy.findByRole("textbox", { name: /Government e-mail/i }).type(
+      "test@tbs-sct.gc.ca",
+    );
 
     cy.findByRole("textbox", {
       name: /What is the job title for this position\?/i,
-    })
-      .type("Test Job Title");
+    }).type("Test Job Title");
 
-    cy.findByRole("textbox", { name: /Additional Comments/i })
-      .type("Test Comments");
+    cy.findByRole("textbox", { name: /Additional Comments/i }).type(
+      "Test Comments",
+    );
 
     cy.findByRole("combobox", { name: /Department/i }).select(
       "Treasury Board Secretariat",
     );
 
-    // will actually exist twice in the DOM
+    // classification filter
+    cy.get("@testClassification1").then((classification) => {
+      cy.findAllByText(classificationName(classification)).should("exist");
```

Comment:
why is findAll* needed here? I'm seeing the code below seem to match the find text as well, but it's hidden. And so I feel like it shouldn't be picked up? so wondering if we're doing something funny that screen-readers will also misread. Seems like it's screen width thing maybe? (a little above the context)

https://github.com/GCTC-NTGC/gc-digital-talent/blob/e0404a71ecbc8b126b64629694f2e3760cf97b4c/frontend/common/src/components/SearchRequestFilters/SearchRequestFilters.tsx#L69-L71

Reply:
This comment applies in a few places where findAllByText is used

Reply:
LOL, there are two separate divs on the page, but only one is visible at a time.
![image](https://user-images.githubusercontent.com/8978655/206549969-a0fb17cf-3cc5-480e-9209-7e2a0f43ee01.png)
@esizer This would need an aria-hidden as well to keep screenreaders from reading twice, right?

Reply:
What is the purpose behind the hidden one? 🤔

Reply:
lol nvm, i see that we switch them out depending on screen size. I don't think `aria-hidden` is necessary because `display: none;` effectively does that. You could remove it from the DOM entirely using the `useIsSmallScreen` instead but that isn't necessary.